core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1082,7 +1082,7 @@ private[kafka] class Processor(
   // `protected` for test usage
   protected[network] def sendResponse(response: RequestChannel.Response, responseSend: Send): Unit = {
     val connectionId = response.request.context.connectionId
-    trace(s"Socket server received response to send to $connectionId, registering for write and sending data: $response")
+    trace(s"Socket server received response from $connectionId, registering for write and sending data: $response")
```

Comment:
why is the suggested comment more correct? we are sending the response _to_ the connection, we are not receiving a response from the connection.

Reply:
Oh, you’re right. Thanks.